### ISSUE TITLE
Update code links in docs to point to the R1.0.0 master branch

### ISSUE THREAD

**Issue by jrobble** [openmpf/openmpf#172]
This task should be worked in tandem with #295.

The "For an example, please see: Hello World JSON Descriptor" link in https://openmpf.github.io/docs/site/Packaging-and-Registering-a-Component/ points to the develop branch. This is because that master branch currently doesn't mention the `supportsBatchProcessing` flag. This is what's in the develop branch:

```
"algorithm" : {
    "name" : "JavaTest",
    "description" : "A dummy Java detection component.",
    "actionType" : "DETECTION",
    "supportsBatchProcessing": true,
```

 Update the doc to point to the master branch once the develop code is merged into the master branch.

Also, please change:

> In order to registered within OpenMPF,

to:

> In order to be registered within OpenMPF,
